src/main.qml

```diff
@@ -51,10 +51,33 @@ Application {
         icon: "ios-sync"
         visible: !btStatus.connected
     }
+
     Item {
         anchors.fill: parent
         visible: btStatus.connected
 
+        Image {
+            id: coverArt
+            anchors.fill: parent
+            fillMode: Image.PreserveAspectCrop
+            opacity: 0.7
+            source: if (mprisManager.currentService) {
```

Comment:
Uh this is a weird syntax. Do you know what happens when the if branch is not taken ? Is there a default return ?